.github/workflows/jacoco_check.yml

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright 2022 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: JaCoCo report
+
+on:
+  pull_request:
+    branches: [ master ]
+    types: [ opened, edited, synchronize, reopened ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - scala: 2.11.12
+            scalaShort: "2.11"
+            spark: 2.4.8
+            overall: 0.0
+            changed: 80.0
+          - scala: 2.12.17
+            scalaShort: "2.12"
+            spark: 3.2.3
+            overall: 0.0
+            changed: 80.0
+          - scala: 2.13.10
+            scalaShort: "2.13"
+            spark: 3.3.2
+            overall: 0.0
+            changed: 80.0
```

Comment:
Since the code for all versions of Scala is the same, we don't need to run the coverage check against the Scala version matrix. Maybe let's keep just 2.13?

Reply:
Code is same, but JaCoCo results are different with changing version of Scala. I would like to keep all project supported combination simply to have insight there.